openedx/features/colaraz_features/api/serializers.py

```diff
@@ -2,69 +2,106 @@
 Serializers for colaraz API.
 """
 from django.conf import settings
+from django.contrib.auth.models import User
 
-from rest_framework import serializers
+from future.moves.urllib.parse import urlparse
 from organizations.models import Organization
+from rest_framework import serializers
 
 from openedx.features.colaraz_features.api.fields import DomainField
 from openedx.features.colaraz_features.helpers import (
-    create_sites, create_site_themes, create_organization, create_site_configurations, do_sites_exists
+    create_oauth2_client_for_ecommerce_site,
+    create_site_on_ecommerce,
+    get_existing_site_domains,
+    get_or_create_organization,
+    get_or_create_sites,
+    update_or_create_site_configurations,
+    update_or_create_site_themes,
 )
 
 
 class SiteOrgSerializer(serializers.Serializer):
     site_domain = DomainField(max_length=90)
     site_name = serializers.CharField(max_length=40, required=False, default='')
-    site_theme = serializers.CharField(max_length=255, default=settings.DEFAULT_SITE_THEME)
+    site_theme = serializers.CharField(max_length=255, required=False, default=settings.DEFAULT_SITE_THEME)
     org_name = serializers.CharField(max_length=255)
-    org_short_name = serializers.SlugField(max_length=255)
     platform_name = serializers.CharField(max_length=255)
+    company_type = serializers.CharField(max_length=255, required=False, default='')
     university_name = serializers.CharField(max_length=255, required=False, default='')
     organizations = serializers.ListField(
         child=serializers.SlugField(max_length=255)
     )
+    site_partner = serializers.CharField(max_length=255, required=False, default='edx')
+    payment_processors = serializers.CharField(max_length=255, required=False, default='')
+    client_side_payment_processor = serializers.CharField(max_length=255, required=False, default='')
```

Comment:
Are we sure that we want 255 characters for these fields?

Reply:
Does it make any difference?